Source/Utils/Notifications.swift

```diff
@@ -110,6 +110,9 @@ extension Notifications {
             let notification = self.notification(Notifications.iceCandidateReceived)
             let userInfo = [NotificationKeys.message.rawValue: iceCandidate]
             self.center.post(name: notification, object: self, userInfo: userInfo)
+
+        case .newChannel:
```

Comment:
Redundant. caller of this method could directly call post(.chatListUpdated), since it has switch anyway